src/scripts/main.js

```diff
@@ -357,4 +357,25 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const peopleTable = document.querySelector('.dashboard tbody');
+
+if (people && Array.isArray(people)) {
+  people.forEach((person) => {
+    const { name: namePerson, born, died } = person;
+    let { sex } = person;
+
+    sex = sex === 'm' ? (sex = 'Male') : (sex = 'Female');
```

Comment:
The reassignment of the `sex` variable can be simplified. Instead of using a ternary operator with an assignment, you can directly assign the result: `sex = sex === 'm' ? 'Male' : 'Female';`. This makes the code cleaner and easier to read.